File: ATDemo(OC)/ATTextView/TextViewBinding/ATTextViewBinding.m
//
//  ATTextViewBinding.m
//  AtDemo
//
//  Created by HN on 2021/4/29.
//

#import "ATTextViewBinding.h"

@implementation ATTextViewBinding

- (instancetype)initWithName:(NSString*)name
                      userId:(NSInteger)userId {
    if (self = [super init]) {
        _name = name;
        _userId = userId;
    }
    return self;
}
@end

File: ATDemo(Swift)/ATDemo/ATTextView/ATTextViewBinding.swift
//
//  ATTextViewBinding.swift
//  ATDemo
//
//  Created by XWH on 2021/5/15.
//

import UIKit
import Foundation

let ATTextBindingAttributeName = "TextViewBingDingFlagName"

class ATTextViewBinding: NSObject {
    var name: String! = ""
    var userId = 0
    var range: NSRange! = NSRange(location: 0, length: 0)

    init(
        name: String?,
        userId: Int
    ) {
        super.init()
            self.name = name
            self.userId = userId
    }
}
